src/config/settings.js

```diff
@@ -177,6 +177,68 @@ export default {
           },
         ]
       },
+      CompanionWindow: {
+        styleOverrides: {
+          closeButton: {
+            order: 4,
+          },
+          contents: {
+            overflowY: 'auto',
+            wordBreak: 'break-word',
+          },
+          controls: ({ ownerState }) => ({
+            alignItems: 'center',
+            display: 'flex',
+            flexFlow: 'row wrap',
+            flexGrow: 1,
+            justifyContent: (ownerState?.position === 'bottom' || ownerState?.position === 'far-bottom') ? 'flex-end' : 'flex-start',
+            minHeight: 48,
+            order: 3
+          }),
+          positionButton: {
+            marginLeft: -16,
+            order: -100,
+            width: 24,
+          },
+          resize: ({ ownerState }) => ({
+            display: 'flex',
+            flexDirection: 'column',
+            minHeight: 50,
+            minWidth: (ownerState?.position === 'left') ? 235 : 100,
```

Comment:
Can you remove the minWidth here if it has been relocated? 

https://github.com/ProjectMirador/mirador/blob/cefe7b7b520ae1cb637d935c7b3cfceaff96b14a/src/components/CompanionWindow.js#L110

Reply:
I don't think so? the react-rnd library wants to know that information too 🤷‍♂️